docs/templates/how-to-template.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Start with a gerund
+content_type: how-to
+no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+beta: true # Labels the page as beta; adds a banner to the top of the page.
+disable_image_expand: true # Stops images from expanding in a modal on click. Sets it for the entire page.
+class: no-copy-code # Disables the copy code button for all code blocks on the page.
+---
+
+<!-- Add an introduction paragraph here. Good introductions explain who this how-to guide is for and what this guide will help the user accomplish. -->
+<!-- Keep the conceptual information to a minimum; we can assume the user already has some knowledge of the concepts. You can also add links to explanation articles if needed. -->
+<!-- See https://documentation.divio.com/how-to-guides/ for more info about how to write a how-to -->
+<!-- See the following examples of how-to documentation:
+* https://docs.konghq.com/gateway/latest/kong-enterprise/analytics/reports/
+* https://docs.konghq.com/gateway/latest/kong-manager/auth/ldap/service-directory-mapping/
+* https://docs.konghq.com/gateway/latest/plugin-development/custom-entities/
+-->
+<!-- Remove these comments once you are done writing -->
+
+## Prerequistes <!-- Optional -->
+
+<!-- Write prerequisites as a bulleted list. Only list prerequisites if they are application level prerequisites. -->
+<!-- If it isn't a "getting started" topic, we can assume our products are installed. -->
+<!-- Don't prescribe Konnect or Gateway role permissions. -->
+
+* First prerequisite
+* Second prerequisite
+
+## Task section <!-- Header optional if there's only one task section in the article -->
+
+<!-- Title must be tasked-based and start with a gerund -->
```

Comment:
Q: Should this be a gerund? To me, "Updating a plugin" implies a wholistic explanation of what "upgrading" is, whereas "upgrade a plugin" is a direction. I've always thought of gerunds as appropriate for explanation/concept topics, whereas how-tos and tutorials should have actions/directions.

Reply:
I don't know what a `gerund` is (or didn't until I just Googled). Can we make this friendlier to contributors?

Reply:
@lena-larionova That makes sense to me, and people will probably search for something like "how to upgrade a Kong plugin" vs. "upgrading a Kong plugin". I'll remove the part about "gerund" and put "verb" there instead.